### ISSUE TITLE
Ctrl+Shift+V crashes Geany

### ISSUE THREAD

**Issue by TheDcoder** [geany/geany#2841]
Hi, I just lost some work because Geany crashed when I accidentally pressed `Ctrl+Shift+V` :sob:

Here is the terminal output:
```
VALUE: 0

(geany:30761): GLib-GObject-WARNING **: 04:09:44.697: ../glib/gobject/gsignal.c:2614: signal 'size-request' is invalid for instance '0x5623a0c66cf0' of type 'GtkComboBox'

** (geany:30761): WARNING **: 04:09:44.701: Error loading plugin: libnuspell.so.5: cannot open shared object file: No such file or directory

fish: Job 1, 'geany' terminated by signal SIGSEGV (Address boundary error
```

I don't know if the issue is caused by some plugin, but it still falls under the geany project umbrella so posting here.

**Comment by elextr** [comment]
Without any information there is nothing we can do, post versions of Geany, GTK, OS, try it with no plugins and lastly run under GDB and get a backtrace when it crashes.  We can't see your system, you may be upset you lost some work, thats understandable, but if you provide no information we can do nothing.  

Also search for existing issues, its a known problem thats not in Geany code and nothing we can do, see #2813.

Closed as duplicate of #2813

**Comment by TheDcoder** [comment]
Right, I was in a bit of hurry last night and I thought the issue was universal (a bit silly in retrospect because someone was bound to notice if it was universal as `Ctrl+Shift+V` is a common combination of keys).

I posted a reply with a suggested hotfix in the non-duplicate issue.
